doc/api/readers/index.rst

```diff
@@ -23,6 +23,7 @@ Reader Classes
    :toctree: _autosummary
    :template: custom-class-template.rst
 
+    
```

Comment:
Sorry for the confusion, the added line had the extra whitespace.  Unless this is needed somehow, let's remove that.

```suggestion
```

Reply:
okay. btw where are mods. One approval is left for commit.